locales/nl/nl.json

```diff
@@ -250,16 +250,16 @@
     "Guinea-Bissau": "Guinee-Bissau",
     "Guyana": "Guyana",
     "Haiti": "Haiti",
-    "Have a coupon code?": "Have a coupon code?",
-    "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.",
-    "Heard Island & Mcdonald Islands": "Heard Eiland en McDonald Eilanden",
-    "Heard Island and McDonald Islands": "Heard Island and McDonald Islands",
+    "Have a coupon code?": "Kortingscode?",
+    "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Twijfelt u over het annuleren van uw abonnement? U kunt uw abonnement op elk moment direct weer activeren tot het eind van het huidige betalingstermijn. Nadat uw huidige abonnement is afgelopen kunt u een ander abonnement kiezen.",
+    "Heard Island & Mcdonald Islands": "Heard Eiland & McDonald Eilanden",
+    "Heard Island and McDonald Islands": "Heard Eiland en McDonald Eilanden",
     "Hello!": "Hallo!",
     "Hide Content": "Inhoud Verbergen",
     "Hold Up!": "Wacht Even!",
-    "Holy See (Vatican City State)": "Vatican City",
+    "Holy See (Vatican City State)": "Vaticaanstad",
     "Honduras": "Honduras",
-    "Hong Kong": "Hong",
+    "Hong Kong": "Hong Kong",
```

Comment:
If it's same translation, it's for exclude/nl.php

Reply:
Should be solved in https://github.com/Laravel-Lang/lang/pull/1733/commits/26f0926779cbdd1cac78503d99940599f81c7178